docs/staking-modules/csm/validator-exits.md

```diff
@@ -0,0 +1,38 @@
+# Validator Exits
+![exits-1](../../../static/img/csm/exits-1.png)
+
+## Voluntary exits
+Given the permissionless nature of CSM, NOs can voluntarily exit their validators at any moment.
+
+## Protocol-initiated exits
+For consistency with the core protocol and other staking modules, CSM uses [VEBO](../../contracts/validators-exit-bus-oracle) to request or trigger exits (to be implemented after the Pectra hardfork bringing [EIP-7002](https://eips.ethereum.org/EIPS/eip-7002) to life) for the validators.
+
+From the core protocol side, validator exit can be requested to cover withdrawal requests from stETH holders, or according to the decision of the DAO.
+
+From CSM side, validator exits can be requested for unbonded validators. These exits are requested automatically using the `forcedTargetLimit`.
+
+> `forcedTargetLimit` is currently under development within the updated version of [Staking Router](https://hackmd.io/@lido/BJXRTxMRp#Forced-Exit-Requests1). In short, it is similar to the existing `targetLimit` but exits for the validators above `forcedTargetLimit` can be requested immediately, even without a need to fulfill withdrawal requests from stETH holders.
```

Comment:
not immediately, but as a part of the nearest VEBO report though

Reply:
Fixed